iina/AutoFileMatcher.swift

```diff
@@ -45,9 +46,11 @@ class AutoFileMatcher {
 
     // group by extension
     for file in files {
-      let fileInfo = FileInfo(file)
-      if let mediaType = Utility.mediaType(forExtension: fileInfo.ext) {
-        filesGroupedByMediaType[mediaType]!.append(fileInfo)
+      autoreleasepool{
```

Comment:
IIRC Chris Lattner had said somewhere that the compiler will detect this and return an object with a +0 retain count in this case.